api-reference/beta/toc.yml

```
@@ -13964,6 +13964,49 @@ items:
               href: api/policyroot-list-rolemanagementpolicyassignments.md
             - name: Get
               href: api/unifiedrolemanagementpolicyassignment-get.md
+          - name: Role management alerts
+            items:
+            - name: Alerts
+              href: resources/unifiedrolemanagementalert.md
+            - name: List
+              href: api/rolemanagementalert-list-alerts.md
+            - name: Get
+              href: api/unifiedrolemanagementalert-get.md
+            - name: Refresh alert
+              href: api/unifiedrolemanagementalert-refresh.md
+            - name: Update alert
+              href: api/unifiedrolemanagementalert-update.md
```

nit: update before refresh
```suggestion
            - name: Update
              href: api/unifiedrolemanagementalert-update.md
            - name: Refresh
              href: api/unifiedrolemanagementalert-refresh.md
```